utils/cloze_utils.py

```diff
@@ -57,9 +57,12 @@ def cloze_output_parser(cloze: str) -> str:
     cloze = cloze.strip()
 
     # make sure all newlines are consistent for now
+    # TODO: You mean <br/>?
```

Comment:
I don't know too much about html, but I think I used some wrong html tags in my notes, so it made sense to be agressive in replacing those. Doesn't hurt? Don't mind if you get rid of them.

Reply:
In html that tries to be valid xhtml tags that don't close (e.g. there isn't a matching tag with a slash before it), the tag has a slash _after_, so `</br>` should never exist.

It doesn't hurt to keep it, I suppose